File: leetcode/python/1347_minimum_number_of_steps_to_make_two_strings_anagram.py
class Solution:
    def minSteps(self, s: str, t: str) -> int:
        cnt = [0] * 26
        for c in s:
            cnt[ord(c) - ord('a')] += 1
        for c in t:
            cnt[ord(c) - ord('a')] -= 1
        return sum([abs(x) for x in cnt]) // 2

File: leetcode/rust/src/1347_minimum_number_of_steps_to_make_two_strings_anagram.rs
impl Solution {
    // Minimum Number of Steps to Make Two Strings Anagram
    pub fn min_steps(s: String, t: String) -> i32 {
        let mut s_bucket = vec![0; 26];
        let mut t_bucket = vec![0; 26];

        for c in s.chars() {
            s_bucket[c as usize - 'a' as usize] += 1;
        }

        for c in t.chars() {
            t_bucket[c as usize - 'a' as usize] += 1;
        }

        let mut res = 0;
        for i in 0..26 {
            if s_bucket[i] > t_bucket[i] {
                res += s_bucket[i] - t_bucket[i];
            }
        }
        res
    }
}